chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsStateTestUtil.kt

```diff
@@ -0,0 +1,48 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.Assert
+import assertk.assertions.containsOnly
+import assertk.assertions.isEmpty
+import assertk.assertions.prop
+
+object JoinedRoomsStateTestUtil {
```

Comment:
it would be more idiomatic to skip the object and have top-level definitions

Reply:
👍